code/modules/power/singularity/emitter.dm

```diff
@@ -401,32 +422,33 @@
 	auto.Remove(buckled_mob)
 	. = ..()
 
-/obj/machinery/power/emitter/prototype/user_buckle_mob(mob/living/M, mob/user, check_loc = TRUE)
+/obj/machinery/power/emitter/prototype/user_buckle_mob(mob/living/mob, mob/user, check_loc = TRUE)
```

Comment:
What is living/mob  in the context of this proc? longer var names are meant to make things more descriptive, not just less messy

Reply:
changed to buckled_mob